lib/src/main/java/io/ably/lib/types/BaseMessage.java

```diff
@@ -173,7 +173,7 @@ public JsonElement serialize(BaseMessage message, Type typeOfMessage, JsonSerial
 			if(message.clientId != null) json.addProperty("clientId", message.clientId);
 			if(message.connectionId != null) json.addProperty("connectionId", message.connectionId);
 			return json;
-		}		
+		}
```

Comment:
Are these due to a change of IDE?

Reply:
Sort of - I saw those as I was looking through the code last night and fixed them. So they were explicit changes, but I noticed them because of a change of IDE.